code/src/nuvla/job/actions/nuvlabox_revoke_ssh_key.py

```diff
@@ -1,24 +1,25 @@
 # -*- coding: utf-8 -*-
 
+import os
 import logging
 import json
 
 from ..actions import action
 from ...connector import nuvlabox as NB
-
+from ...connector.kubernetes import K8sSSHKey
 
 @action('nuvlabox_revoke_ssh_key', True)
 class NBRevokeSSHKey(object):
-
+    '''Here'''
     def __init__(self, _, job):
         self.job = job
         self.api = job.api
 
     def revoke_ssh_key(self):
+        '''Here'''
```

Comment:
?

Reply:
Removed the doc string to conform to how other do_work are handled